tests/test_manifest.py

```diff
@@ -393,7 +393,7 @@ def test_project_revisions():
     ''')
     expected = [Project('p1', 'u1', revision='defaultrev'),
                 Project('p2', 'u2', revision='rev')]
-    for p, e in zip(m.projects[1:], expected):
+    for p, e in zip(m.projects[1:], expected, strict=True):
```

Comment:
Can you explain how this is related to the Python version? I find this one hard to guess...